go.mod

```diff
@@ -1,36 +1,27 @@
 module sigs.k8s.io/kind
 
 require (
-	github.com/emicklei/go-restful v2.8.0+incompatible // indirect
-	github.com/evanphx/json-patch v3.0.0+incompatible // indirect
+	github.com/evanphx/json-patch v4.2.0+incompatible // indirect
 	github.com/ghodss/yaml v1.0.0 // indirect
-	github.com/go-openapi/jsonpointer v0.17.2 // indirect
-	github.com/go-openapi/jsonreference v0.17.2 // indirect
-	github.com/go-openapi/spec v0.17.2 // indirect
-	github.com/go-openapi/swag v0.17.2 // indirect
-	github.com/gogo/protobuf v1.1.1 // indirect
-	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf
+	github.com/go-openapi/spec v0.19.0 // indirect
+	github.com/gogo/protobuf v1.2.1 // indirect
+	github.com/google/gofuzz v1.0.0
 	github.com/google/uuid v1.1.1
-	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/json-iterator/go v1.1.5 // indirect
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 // indirect
-	github.com/onsi/ginkgo v1.7.0 // indirect
-	github.com/onsi/gomega v1.4.3 // indirect
```

Comment:
i'm really not sure how these were still in the root go.mod (ginkgo, gomega), happy to have the dependencies reduced, it should make building faster :-)